praw/__init__.py

```diff
@@ -986,6 +986,23 @@ def get_submissions(self, fullnames, *args, **kwargs):
             for item in self.get_content(url, limit=len(cur), *args, **kwargs):
                 yield item
 
+    def get_comment(self, comment_id):
+        """Return a Comment object for the given comment_id"""
+        url=None
```

Comment:
What does `url` represent? It appears that it represents if there is more than one object.